src/Orleans.Hosting.ServiceFabric/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Fabric;
+using System.Fabric.Description;
+using System.Net;
+using Orleans.ServiceFabric;
+using NodeConfiguration = Orleans.Runtime.Configuration.NodeConfiguration;
+
+namespace Orleans.Hosting
+{
+    public static class ConfigurationExtensions
```

Comment:
Please use a class name that has `ServiceFabric` in it. Especially since this goes to the default Orleans.Hosting namespace where collisions are likely to occur

Reply:
good call, will fix